cloudburst/server/scheduler/server.py

```diff
@@ -124,7 +124,12 @@ def scheduler(ip, mgmt_ip, route_addr):
     pin_accept_socket.bind(sutils.BIND_ADDR_TEMPLATE %
                            (sutils.PIN_ACCEPT_PORT))
 
-    requestor_cache = SocketCache(context, zmq.REQ)
+    management_request_socket = context.socket(zmq.REQ)
+    management_request_socket.setsockopt(zmq.RCVTIMEO, 500)
+    management_request_socket.setsockopt(zmq.REQ_CORRELATE, 1)
```

Comment:
Can we explain what these parameters do?

Reply:
yes